browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -116,6 +117,12 @@ bool IsAvatarButtonHideable(Profile* profile) {
   return !profile->IsIncognitoProfile() && !profile->IsGuestSession();
 }
 
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+bool BraveVpnHasConnectionAPI() {
```

Comment:
This method name doesn't really make sense to me, but either way the BraveToolbarView shouldn't have to know that it needs to check for null `g_brave_browser_process->brave_vpn_connection_manager()` when it doesn't even use it directly so I think this needs to be handled a different way

Reply:
Reverted this and added [eff649f](https://github.com/brave/brave-core/pull/20433/commits/eff649fe08021a1eceeed9e7a6b067aa0db60dc4)